pkg/noderesourcetopology/filter_test.go

```diff
@@ -424,8 +424,8 @@ func TestNodeResourceTopologyMultiContainerPodScope(t *testing.T) {
 	}{
 		{
 			name: "gu pod fits only on a numa node",
-			pod: makeMultiContainerPodWithResourceList("testpod",
-				[]v1.ResourceList{
+			pod: makePod("testpod",
+				withMultiContainers([]v1.ResourceList{
```

Comment:
This commit includes refactoring and addition of test cases, I would split this commit into two to separate those out for better clarity.

Reply:
Sure. I'll work on it